spec/support/active_record.rb

```diff
@@ -1,6 +1,12 @@
-require "support/active_record"
 require "json"
 
+MIGRATION_CLASS = if Rails.version.split(".").map(&:to_i).first >= 5
+                    migration_version = ActiveRecord::Migration.current_version
+                    ActiveRecord::Migration[migration_version]
+                  else
+                    ActiveRecord::Migration
+                  end
+
```

Comment:
Similarly here - I'd rather either:
- update all the specs to use AR 5.0 syntax and have a single explicit test for AR < 5.0
- keep all the specs on AR 4.0 < 5.0 syntax and have an explicit test for AR 5.0